policies/api/grpc/server.go

```diff
@@ -26,6 +27,7 @@ type grpcServer struct {
 	pb.UnimplementedPolicyServiceServer
 	retrievePolicy           kitgrpc.Handler
 	retrievePoliciesByGroups kitgrpc.Handler
+	inactivateDataset        kitgrpc.Handler
```

Comment:
changes in this file can be reverted right? since this is no longer a gRPC method